task-1/results/dags/connections.py

```
@@ -0,0 +1,35 @@
+from airflow.models import Connection
+
+def create_connections():
+    """Создание подключений к базам данных"""
+
+    new_conn = Connection(
+        conn_id='postgres_data',
+        conn_type='postgres',
+        host='postgres-data',
+        schema='datadb',
+        login='datauser',
+        password='datapass',
```

Ruff (0.13.1)

Possible hardcoded password assigned to argument: "password"

(S106)